sass/components/_sword.scss

```diff
@@ -0,0 +1,51 @@
+.sword {
+  height: 50px;
+  width: 1280px;
```

Comment:
a próbowałaś może najpierw zrobić width i height z procentowymi wartościami? Bo to od razu super by się skalowało do responsywności :-) i nie trzeba by było grzebać przy tym (przynajmniej tak mi się wydaje ^_^ )

Reply:
@PiotrWR Ty jesteś chyba naszym specem od CSS'a ... co sądzisz?

Reply:
@DomiZet , @Szambelan 
Super, że to zauważyłeś! Dokładnie tak, wartości procentowe i będzie elegancko!
Widzę, że dokładne CR zrobiłeś :)